FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,48 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
+		client.get(from: url) { [weak self] result in
+			guard self != nil else {return}
```

Comment:
Please add spacings around `return` to follow the code convention of the codebase

```suggestion
			guard self != nil else { return }
```